plugins/auto-sizes/readme.txt

```diff
@@ -1,8 +1,8 @@
 === Auto-sizes for Lazy-loaded Images ===
 
 Contributors:      wordpressdotorg
-Requires at least: 6.4
-Tested up to:      6.5
+Requires at least: 6.5
+Tested up to:      6.6
```

Comment:
FWIW, "Requires at least" and "Requires PHP" headers should only be defined in the main plugin file, as that's where the information is extracted from in the plugin directory. It's redundant to have it in `readme.txt`.

Reply:
I was wondering about that, and I was going to remove `Requires at least` from the readme, but then I saw it was shown in an example in the docs: https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information

But I skipped over this line:

> Since WordPress 5.8 plugin [readme files are not parsed for requirements](https://core.trac.wordpress.org/ticket/48520). This means that headers Requires PHP and Requires at least are going to be parsed from plugin’s main PHP file.

Reply:
I've opened https://github.com/WordPress/performance/pull/1334 to address this.